web/src/pages/news/blogs/index.page.tsx

```diff
@@ -1,11 +1,123 @@
+import { NextSeo } from "next-seo";
 import React from "react";
+import { StoryblokStory } from "storyblok-generate-ts";
 
-export const BlogListingPage = (): React.ReactElement => {
+import { ActionBanner } from "@nice-digital/nds-action-banner";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Button } from "@nice-digital/nds-button";
+import { PageHeader } from "@nice-digital/nds-page-header";
+
+import { ErrorPageContent } from "@/components/ErrorPageContent/ErrorPageContent";
+import { FeaturedStory } from "@/components/Storyblok/News/FeaturedStory/FeaturedStory";
+import { NewsList } from "@/components/Storyblok/News/NewsList/NewsList";
+import { NewsListNav } from "@/components/Storyblok/News/NewsListNav/NewsListNav";
+import { NewsListPagination } from "@/components/Storyblok/News/NewsListPagination/NewsListPagination";
+import { NewsStory } from "@/types/News";
+import { validateRouteParams } from "@/utils/storyblok";
+
+import type { GetServerSidePropsContext } from "next";
+
+export type BlogPostsProps = {
+	featuredStory?: StoryblokStory<NewsStory> | null;
+	stories: StoryblokStory<NewsStory>[];
+	total: number;
+	currentPage: number;
+	perPage: number;
+	error?: string | undefined;
+};
+
+const destinations = [
```

Comment:
These are going to be the same on every news page - thinking that maybe we should hard-code this list in the NewsListNav component otherwise they'll get repeated for each news page which isn't very DRY. Probably one for the tech debt pile though, as it'll need updating in all pages/components in one go